__tests__/setup/createDirectLineForwarder.js

```diff
@@ -0,0 +1,25 @@
+import { createProxyServer } from 'http-proxy';
+import { createServer } from 'http';
+import { promisify } from 'util';
+
+export default async function createDirectLineForwarder(port, handler, target = 'https://directline.botframework.com/') {
+  // We need a reverse proxy (a.k.a. forwarder) to control the network traffic.
+  // This is because we need to modify the HTTP header by changing its host header (directline.botframework.com do not like "Host: localhost").
+
+  const proxy = createProxyServer({
+    changeOrigin: true,
+    rejectUnauthorized: false,
```

Comment:
don't need this line?

Reply:
Figured out we still need this one if we are using Fiddler to hack/debug the traffic.

Reply:
Figured out we still need this line `rejectUnauthorized: false` because the upstream of this proxy could be Fiddler, which it will use a self-signed certificate.